pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/Event.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.dsm.util;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class Event {
```

Comment:
With all the code in the same repo now, instead of duplicating json wrapper classes in client and server code, we should re-use what we already have.  Take a look at org.broadinstitute.ddp.json.dsm.DsmNotificationPayload.

Reply:
I moved the `DsmNotificationPayload` and relevant classes from dss-core to ddp-common package (which has all of the classes that both dss and dsm use and) and changed the code to use those classes.